css/types/global_keywords.json

```diff
@@ -209,6 +209,55 @@
             }
           }
         },
+        "revert_layer": {
```

Comment:
We don't replace hyphens with underscores; this allows us to provide less conversion work for consumers!

```suggestion
        "revert-layer": {
```